signac/contrib/job.py

```diff
@@ -484,20 +493,20 @@ def document(self):
             persistent JSON file, use the call operator to get an equivalent
             plain dictionary: ``job.document()``.
             For more information, see
-            :class:`~signac.synced_collections.collection_json.BufferedJSONDict`.
+            :class:`~signac.synced_collections.collection_json.BufferedJSONAttrDict`.
 
         See :ref:`signac document <signac-cli-document>` for the command line equivalent.
 
         Returns
         -------
-        :class:`~signac.JSONDict`
+        :class:`~signac.BufferedJSONAttrDict`
```

Comment:
If we import the `BufferedJSONAttrDict as JSONDict` above, then we should revert to the old name `signac.JSONDict`.